src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -34,13 +34,13 @@ namespace Microsoft.IdentityModel.Tokens
     /// Creates <see cref="SignatureProvider"/>s by specifying a <see cref="SecurityKey"/> and algorithm.
     /// <para>Supports both <see cref="AsymmetricSecurityKey"/> and <see cref="SymmetricSecurityKey"/>.</para>
     /// </summary>
-    public class SignatureProviderFactory
+    public class CryptoProviderFactory
```

Comment:
We plan to add JWE and add methods to create encryption provider later. We are trying to update the OM so that JWE can be added later without breaking stuff.